CODEOWNERS

```diff
@@ -0,0 +1,22 @@
+# Lists some code owners.
+#
+# A codeowner just oversees some part of the codebase. If an owned file is changed then the
+# corresponding codeowner receives a review request. An approval of the codeowner might be
+# required for merging a PR (depends on repository settings).
+#
+# For details about syntax, see:
+# https://help.github.com/en/articles/about-code-owners
+# But here are some important notes:
+#
+# - Glob syntax is git-like, e.g. `/core` means the core directory in the root, unlike `core`
+#   which can be everywhere.
+# - Multiple owners are supported.
+# - Either handle (e.g, @github_user or @github_org/team) or email can be used. Keep in mind,
+#   that handles might work better because they are more recognizable on GitHub,
+#   you can use them for mentioning unlike an email.
+# - The latest matching rule, if multiple, takes precedence.
+
+# CI
+/.github/ @paritytech/ci
+/scripts/ @paritytech/ci @joao-paulo-parity
```

Comment:
The /scripts folder is used for internal needs of the project, not CI scripts. So I'd remove this line altogether.

Reply:
It can also be executed locally. In fact I often do execute locally before pushing. I'd insist on removing this from CODEOWNERS, it does not make sense for the CI team to be notified for changes on this file.

Reply:
Removed `scripts` folder from CODEOWNERS